source/opt/instruction.h

```diff
@@ -35,6 +35,19 @@ class IRContext;
 class Module;
 class InstructionList;
 
+// Relaxed logcial addressing:
+//
+// In the logical addressing model, pointers cannot be stored or loaded.  This
+// is a useful assumption because it simplifies the aliasing significantly.
+// However, for the purpose of legalizing HLSL type code, we will have to allow
+// storing and loading of pointers to opaque objects and runtime arrays.  This
+// relaxation of the rule still implies that function and private scope
+// variables do not have any aliasing, so we can treat them as before.  This
+// will be cased the relaxed logical addressing model.
```

Comment:
"will be cased" ---> "will be the case in"

Reply:
I meant "will be called".  Changed to that.